stories/topics/con-gcp-upgrade-restore-from-stack.adoc

```diff
@@ -0,0 +1,81 @@
+[id="con-gcp-upgrade-restore-from-stack"]
+
+= Restoring the stack from backups
+
```

Comment:
If, for some reason, the upgrade process fails, you can use the backup files from xref:???? to restore your stack.